sdks/typescript/package.json

```diff
@@ -38,7 +34,15 @@
     "eslint-plugin-jsx-a11y": "^6.7.1",
     "eslint-plugin-prettier": "^5.1.3",
     "eslint-plugin-react": "^7.33.2",
-    "eslint-plugin-react-hooks": "^4.6.0"
+    "eslint-plugin-react-hooks": "^4.6.0",
+    "jest": "^29.7.0",
+    "ts-jest": "^29.1.4",
+    "typescript": "^5.4.5"
   },
-  "main": "dist/index.js"
+  "main": "./lib/cjs/index.js",
+  "module": "./lib/esm/index.js",
+  "types": "./lib/esm/index.d.ts",
+  "dependencies": {
+    "axios": "^1.7.2"
```

Comment:
@danielfernandes2022 as aligned if you could remove `axios` references. Thank you very much.

Reply:
yes :)